docs/windows-nt.md

```diff
@@ -0,0 +1,163 @@
+## Windows NT 3.1
```

Comment:
I'd add a `skip to: XP / 2000 / Vista` here

Reply:
I added small navigation in https://github.com/copy/v86/pull/1206/commits/f87f41bfea9c321bd7efa312468c3e128d9bfb50, however, GitHub has an "Outline" panel for navigating by headings:

![gh-outline](https://github.com/user-attachments/assets/f575e269-c807-4df2-9a69-b855322a4958)